cards/ecg-right-posterior-leads/card.md

```diff
@@ -52,3 +52,6 @@ Hints of an associated posterior infarct:
 2.  Upright T wave waves (in later stage)
 
 ## References
+
+1. [Edhouse J, Brady WJ, Morris F. ABC of clinical electrocardiography: Acute myocardial infarction-Part II. BMJ. 2002; 324: 963-6.] (https://doi.org/10.1136/bmj.324.7343.963)
```

Comment:
This requirement can be changed if you don't like it. A "tab" in this case is 3 spaces, not an actual tab. And technically speaking, the requirement is a tab-stop, not a full tab (details below).

The reason for it is to make sure that numbered list items stay aligned. 

Here's an example without it...

```
[ ... ]
8. List item 8
9. List item 9.
10. List item 10.
```

Here's an example with it...

```txt
[ ... ]
8.  List item 8.
9.  List item 9.
10. List item 10.
```